src/app/report-rollup/fsat-rollup/fsat-rollup.component.ts

```diff
@@ -47,7 +47,8 @@ export class FsatRollupComponent implements OnInit {
   setBarChartOption(str: string) {
     this.barChartDataOption = str;
     if (this.barChartDataOption == 'energy') {
-      this.yAxisLabel = 'Annual Energy Usage (' + this.settings.powerMeasurement + ')';
+      //change here
```

Comment:
All // change here comments can be removed. We'll usually save comments for times where something needs to be explained, why, or rare cases.